CMakeLists.txt

```diff
@@ -1261,7 +1125,6 @@ if(ZLIB_ENABLE_TESTS)
 
     add_simple_test_executable(deflate_quick_bi_valid)
     add_simple_test_executable(deflate_quick_block_open)
-    add_simple_test_executable(hash_head_0)
```

Comment:
This edit seems out-of-place... Has nothing to do with optimizations.

Reply:
Fixed.

Reply:
Fixed.